contracts/governor/GovernorBravoInterfaces.sol

```diff
@@ -0,0 +1,208 @@
+// SPDX-License-Identifier: Apache-2.0
+// Compound https://github.com/compound-finance/compound-protocol/blob/master/contracts/Governance/GovernorBravoInterfaces.sol
+pragma solidity ^0.8.0;
+
+contract GovernorBravoEvents {
+    /// @notice An event emitted when a new proposal is created
+    event ProposalCreated(
+        uint256 id,
+        address proposer,
+        address[] targets,
+        uint256[] values,
+        string[] signatures,
+        bytes[] calldatas,
+        uint256 startBlock,
+        uint256 endBlock,
+        string description
+    );
+
+    /// @notice An event emitted when a vote has been cast on a proposal
+    /// @param voter The address which casted a vote
+    /// @param proposalId The proposal id which was voted on
+    /// @param support Support value for the vote. 0=against, 1=for, 2=abstain
+    /// @param votes Number of votes which were cast by the voter
+    /// @param reason The reason given for the vote by the voter
+    event VoteCast(
+        address indexed voter,
+        uint256 proposalId,
+        uint8 support,
+        uint256 votes,
+        string reason
+    );
+
+    /// @notice An event emitted when a proposal has been canceled
```

Comment:
question: why use the Doxygen tag on the `event` elements, I didn't think they were picked up on them?